### PR TITLE
Add shutdown_timeout parameter

### PR DESCRIPTION
**Pull request by wardhus**
#### Pull Request (PR) description
Add's shutdown_timeout parameter

### CODE REVIEW

**Comment by tuxmea** [comment]
REFERENCE.md is outdated did you use the latest version of puppet-syntax?

File: REFERENCE.md [hunk 1]
```diff
@@ -145,7 +146,15 @@ Default value: `undef`
 
 Data type: `Optional[Integer]`
 
-defines the interval length, in seconds, between new jobs check. The default value is 3; if set to 0 or lower, the default value will be used.
+Defines the interval length, in seconds, between the runner checking for new jobs. The default value is 3. If set to 0 or lower, the default value is used.
```
Comment (smortex) [inline REFERENCE.md:149]: This change does not appear in `manifests/init.pp`, maybe this change was not committed on your side.

CI ensure REFERENCE.md is up-to-date so both files should match.  Please check your local repo, update your commits and push to update the PR.

**Comment by wardhus** [comment]
Just fixed this. I didn't run rake, just edited it manually.  Now i did it with rake.

**Comment by bastelfreak** [comment]
thanks for the PR!

**Pull request merged**